README.md

```diff
@@ -1 +1,47 @@
-# rustybeer
\ No newline at end of file
+# RustyBeer #
+
+RustyBeer is a CLI tool written in Rust, to calculate values used in the process of brewing beer.
+
+## Installation ##
+
+If you don't already have the toolset installed, you will first need to install [Rust](https://doc.rust-lang.org/book/ch01-01-installation.html).
+From the root of the repository, run the following command:
+```shell
+cargo build
+```
+You can now run it with:
+```shell
+./target/debug/rustybeer <subcommand>
```

Comment:
We should also publish this to crates.io and add installation instructions for that. But that can be done later; just FYI: https://doc.rust-lang.org/cargo/reference/publishing.html